data_hub_api/docmaps/v1/codecs/docmaps.py

```diff
@@ -3,6 +3,7 @@
 from typing import Dict, Iterable, Sequence, Union, cast
 import urllib
 
+from data_hub_api.utils.url import get_basepath
```

Comment:
We don't need to update this version of the DocMaps API as we are planning to remove it and don't plan to support it anymore.

Reply:
That's fine but our approach is backwards compatible. If you're happy with the approach then let's get this desployed to staging. I will merge in now. Please let me know when it is available in staging.